bin/validate.py

```diff
@@ -180,3 +180,79 @@ def run(self, testcase, run=None, constraints=None, args=None):
                 cwd=run.feedbackdir,
                 timeout=config.get_timeout(),
             )
+
+# Testcase input is only allowed to contain newlines and printable characters
+def _is_invalid_input_byte(byte):
```

Comment:
maybe change this and the next function from `_is*` to `_has*` and make it take the string itself.

Reply:
i would prefer to not try to decode the bytes :)